### DISCUSSION TITLE
Errno::ENOENT: No such file or directory @ rb_sysopen - /config/signing.key

### DISCUSSION THREAD

**Discussion by faheemkhan5** [postalserver/postal#2975] [Help with using Postal]
Please help me, i have installed the fresh postal mail server v3. i am getting the following error when i try to send an email and the email went to queue with the following error

Errno::ENOENT: No such file or directory @ rb_sysopen - /config/signing.key
![image](https://github.com/postalserver/postal/assets/40638668/f4e764d7-5e17-4f5a-9563-9759a3e94fc7)

**Comment by willpower232** [comment]
Did you `postal bootstrap` at any point? https://docs.postalserver.io/getting-started/installation#configuration

**Reply by faheemkhan5** [reply]
Thank you for your response. i re-install the server with postal bootstrap and it works!!!!!!
